_posts/2020-02-18-metal3-dev-env-install-deep-dive.md

```diff
@@ -118,7 +118,7 @@ Also, `common.sh` library also sets more global variables apart from the those i
 | IRONIC_IMAGE             | quay.io/metal3-io/ironic                                  |
 | IRONIC_INSPECTOR_IMAGE   | quay.io/metal3-io/ironic-inspector                        |
 | BAREMETAL_OPERATOR_IMAGE | quay.io/metal3-io/baremetal-operator                      |
-| CAPI_VERSION             | v1alpha1                                                  |
+| CAPM3VERSION             | v1alpha3                                                  |
```

Comment:
Should this be `CAPM3_VERSION` like it is below?

```suggestion
| CAPM3_VERSION             | v1alpha3                                                  |
```

Reply:
yes indeed, sorry for the typo